cli/ssh.go

```diff
@@ -594,6 +626,20 @@ func (r *RootCmd) ssh() *serpent.Command {
 			Default:     "5s",
 			Value:       serpent.DurationOf(&networkInfoInterval),
 		},
+		{
+			Flag:          "container",
+			FlagShorthand: "c",
+			Description:   "Specifies a container inside the workspace to connect to.",
+			Value:         serpent.StringOf(&container),
+			Hidden:        true, // Hidden until this features is at least in beta.
+		},
+		{
+			Flag:          "container-user",
+			FlagShorthand: "u",
```

Comment:
I'm not sure what we'd use `-u` for otherwise, but allocating `-u` for "container" specific options seems unfortunate. Do we need the shorthand flag at all? Isn't giving a container user manually a rare occurance?

Reply:
I suppose so! I can keep it long-form only for now.